scripts/start.json

```diff
@@ -20,8 +20,8 @@
         },
         {
             "id": "/funcatron/tron",
-            "cpus": 2.0,
-            "mem": 2048,
+            "cpus": 1.0,
```

Comment:
@dpp: since you said I should reduce these in the Kube specs, I also reduced them for Mesos.